test/remote/gateways/remote_checkout_v2_test.rb

```diff
@@ -583,6 +595,36 @@ def test_successful_credit
     assert_equal 'Succeeded', response.message
   end
 
+  def test_successful_store
+    response = @gateway_api.store(@credit_card, @options)
+    assert_success response
+    assert_equal 'Succeeded', response.message
+  end
+
+  def test_failed_store_oauth
+    response = @gateway_oauth.store(@credit_card, @options)
+    assert_failure response
+    assert_equal '401: Unauthorized', response.message
+  end
+
+  def test_successful_unstore_after_store
+    store = @gateway_api.store(@credit_card, @options)
+    assert_success store
+    assert_equal 'Succeeded', store.message
+    source_id = store.params['id']
+    response = @gateway_api.unstore(source_id, @options)
+    assert_success response
+    assert_match "The instrument #{source_id} has bein deleted/unstored", response.message
+  end
+
+  def test_successful_unstore_after_purchase
```

Comment:
💬 Comment:

If this is already working like this, we need to raise our hand and ask if is OK, is weird to store thhe payment method by default.